assets/src/js/admin/reports/components/mini-chart/utils/index.js

```diff
@@ -47,14 +52,64 @@ export function createConfig( data ) {
 				} ],
 				xAxes: [ {
 					display: false,
+					type: 'time',
 				} ],
 			},
 			tooltips: {
+				// Disable the on-canvas tooltip
 				enabled: false,
+				mode: 'index',
+				intersect: false,
+				custom: function( tooltipModel ) {
```

Comment:
Let's make sure this is turned into a portal component, as we discussed a while back.

Reply:
I agree, and opened a separate issue to tackle this functionality. Ref: https://github.com/impress-org/givewp/issues/4494